### DISCUSSION TITLE
What is the equivalent of (Call Shell) in WPF?

### DISCUSSION THREAD

**Discussion by VahidEra** [dotnet/wpf#7530] [Q&A] [answered]
Hi guys,
I used to use **_Call Shell_** to call the location of a file in explorer and in winform like this:

![Animation3](https://user-images.githubusercontent.com/27216710/218425554-e9fc0edf-4ace-431b-8e21-0d71c031cc4d.gif)

By this code:
```
        Call Shell("explorer /select," & File Path, AppWinStyle.NormalFocus)
```

>**_Now how can I do this in WPF?
  I have no idea how to do this_**

thank you very much to helping me.

**Comment by miloush** [comment, marked as answer]
Hmm I don't think that is WinForms specific, the same call should work in WPF too: https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.interaction.shell?view=net-7.0

Are you getting any errors?

**Reply by VahidEra** [reply]
@miloush Dear Jan, You are exactly right. The problem was solved by changing the ```call shell```  to  ```Intersection.shell``` Thank you so much for always solving my problem. Thank you very much.
